### PR TITLE
Open map at user's location

### PR DESCRIPTION
**Pull request by invisiblek**
https://github.com/AHAAAAAAA/PokemonGo-Map/issues/2628

### CODE REVIEW

**Comment by invisiblek** [comment]
toggle added

File: templates/map.html [hunk 1]
```diff
@@ -43,12 +43,12 @@
...
         <div class="form-control switch-container" style="display:{{is_fixed}}" >
           <label for="next-location">
-            <h3>Change your location</h3>
+            <h3>Change scan location</h3>
             <input id="next-location" type="text" name="next-location" placeholder="Change your location">
           </label>
         </div>
         <div class="form-control switch-container" style="display:{{is_fixed}}" >
-          <h3>Follow location</h3>
+          <h3>Scan follow location</h3>
```
Comment (Bitti09) [inline templates/map.html]: doesn't

> Scan follows location


sound better?

Comment (invisiblek) [inline templates/map.html]: Nothing sind good here... Yours is better though

File: templates/map.html [hunk 2]
```diff
@@ -57,6 +57,16 @@
 	    </label>
 	  </div>
 	</div>
+        <div class="form-control switch-container" >
+          <h3>Start at user's location</h3>
+	  <div class="onoffswitch">
```
Comment (andrewda) [inline templates/map.html]: Small indentation issue here and on lines 65/66. Other than that LGTM :+1:

Comment (invisiblek) [inline templates/map.html]: omg

not tabs OR spaces
tabs AND spaces

i have lost the will to live

**Comment by Slivo-fr** [comment]
You should remove the toggle if the -fl flag is used

**Comment by invisiblek** [comment]
@Slivo-fr this doesn't actually change the scanner location (which is what -fl is there to prevent). This simply adjusts the starting point of the map if the user chooses to have it on.

**Comment by Slivo-fr** [comment]
What do you mean starting point ? Does it change the marker location ?
Does the scan happens elsewhere than the location used when starting the python script ?

**Pull request merged**